src/test/java/ru/yandex/qatools/embed/postgresql/TestPostgresStoredData.java

```diff
@@ -0,0 +1,88 @@
+package ru.yandex.qatools.embed.postgresql;
+
+import org.junit.*;
+import org.junit.runners.MethodSorters;
+import ru.yandex.qatools.embed.postgresql.config.AbstractPostgresConfig;
+import ru.yandex.qatools.embed.postgresql.config.PostgresConfig;
+import ru.yandex.qatools.embed.postgresql.distribution.Version;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static ru.yandex.qatools.embed.postgresql.EmbeddedPostgres.*;
+import static ru.yandex.qatools.embed.postgresql.util.SocketUtil.findFreePort;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TestPostgresStoredData {
```

Comment:
Yes, but it would much better to be able to bypass this directory to the `PostgresConfig`,  which is created [here](https://github.com/yandex-qatools/postgresql-embedded/blob/master/src/main/java/ru/yandex/qatools/embed/postgresql/EmbeddedPostgres.java#L124)